src/shared/components/pictrs-image.tsx

```diff
@@ -71,4 +73,11 @@ export class PictrsImage extends Component<PictrsImageProps, any> {
 
     return out;
   }
+
+  alt(): string {
+    if (this.props.icon) {
+      return '';
+    }
+    return this.props.alt || '';
+  }
```

Comment:
I'm not sure I understand this logic. If its an icon, it doesn't need an alt? I thought all pictures needed an alt.

Reply:
Icons are decorative; see the novel I wrote in the other comment